lib/compiler/semantic.js

```diff
@@ -21,6 +23,8 @@ var SemanticPass = Base.extend({
 
     initialize: function(options) {
         this.now = options.now;
+        this.beginning = moment(-Infinity);
```

Comment:
et tu juttle?
per today's discussions over on moment-parser, this creates an invalid moment with NaN as its value.